spoon-client/src/main/java/com/squareup/spoon/Spoon.java

```diff
@@ -206,7 +205,7 @@ private static File filesDirectory(Context context, String directoryType, String
     File directory;
     if (Build.VERSION.SDK_INT >= 21) {
       // Use external storage.
-      directory = new File(getExternalStorageDirectory(), "app_" + directoryType);
+      directory = new File(context.getExternalFilesDir(null), "app_" + directoryType);
```

Comment:
Actually, I missed that. It won't work.

The new path looks like this:

/storage/emulated/0/Android/data/com.joelapenna.foursquared/files/app_spoon-screenshots/com.joelapenna.foursquared.TopPicksTest/goToVenue/1453506199826_goToVenue_end.png

I'm not sure if the format `/storage/emulated/0/Android/data/package_name/app_spoon-screenshots/test_class_name/filename` will be consistent.